File: Servidor/grapeguardapi/src/main/java/com/raquelbytes/grapeguardapi/Model/Usuario.java
package com.raquelbytes.grapeguardapi.Model;

import jakarta.persistence.*;
import java.util.List;
import lombok.Data;

@Entity
@Data
@Table(name = "Usuario")
public class Usuario {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column
    private Integer ID_usuario;
    @Column
    private String nombre;
    @Column
    private String apellido;
    @Column
    private String email;
    @Column
    private String contrasena;
    @Column
    private String foto;
}

File: Movil/GrapeGuard/app/src/main/java/com/raquelbytes/grapeguard/API/Model/Usuario.kt
package com.raquelbytes.grapeguard.API.Model

import java.io.Serializable

class Usuario : Serializable {
    var id_usuario: Int? = null
    var nombre: String? = null
    var apellido: String? = null
    var email: String? = null
    var contrasena: String? = null
    var foto: String? = null

    constructor()

    constructor(id_usuario: Int, nombre: String, apellido: String, email: String, contrasena: String, foto: String) {
        this.id_usuario = id_usuario
        this.nombre = nombre
        this.apellido = apellido
        this.email = email
        this.contrasena = contrasena
        this.foto = foto
    }

    constructor(nombre: String, apellido: String, email: String, contrasena: String, foto: String) {
        this.nombre = nombre
        this.apellido = apellido
        this.email = email
        this.contrasena = contrasena
        this.foto = foto
    }
}
